### PR TITLE
Added SMSCentral channel type

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: temba/settings_common.py [hunk 1]
```diff
@@ -652,6 +654,7 @@
         'channels.channel_claim',
         'channels.channel_claim_africas_talking',
         'channels.channel_claim_android',
+        'channels.channel_claim_blackmyna',
```
Comment (rowanseymour) [inline temba/settings_common.py]: This was missing previously - assuming it was a mistake

**Comment by norkans7** [comment]
You can add the status endpoint for delivery reports

**Comment by rowanseymour** [comment]
Evan checked with them and they don't do delivery reports. They don't even return an external id when you send so you wouldn't be able to know which message to update.

File: temba/channels/models.py [hunk 2]
```diff
@@ -106,6 +108,7 @@
     PLIVO: dict(scheme='tel', max_length=1600),
     HIGH_CONNECTION: dict(scheme='tel', max_length=320),
     BLACKMYNA: dict(scheme='tel', max_length=1600),
+    SMSCENTRAL: dict(scheme='tel', max_length=1600)
```
Comment (ericnewcomer) [inline temba/channels/models.py]: Do we know if they do message splitting?

Comment (rowanseymour) [inline temba/channels/models.py]: We don't know that... pinging Evan...

**Comment by ericnewcomer** [comment]
Ya, aside from determining whether our max_length is right, this looks good to me.

File: temba/channels/models.py (hunk 2 above)
Comment (ewheeler) [inline temba/channels/models.py]: Yes. They say that they will split messages

**Pull request merged**